package.json

```diff
@@ -1,10 +1,10 @@
 {
-  "name": "aeproect",
+  "name": "aeproject",
   "version": "1.0.0",
   "description": "This is the default package.json generated for your project",
   "main": "index.js",
   "scripts": {
-    "test": "mocha './test/**/*.js' --timeout 0 --exit"
+    "test": "mocha './test/commands-tests/*.js' --timeout 0 --exit"
```

Comment:
remove commands-tests, the path should be **./test/**/*.js**

Reply:
Done